react.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
             extensions: ['.js'],
         }],
         'react/jsx-first-prop-new-line': ['error', 'multiline'],
-        'react/jsx-handler-names': 'error',
+        'react/jsx-handler-names': 'off',
```

Comment:
I'm wondering if there's a combination of the rules we can use: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md